example/example.spec.js

```diff
@@ -0,0 +1,77 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+const path = require('path');
+
+const getTextContent = async (element) => {
+	return await element.evaluate((el) => el.textContent);
+};
+
+test.describe('Tram-Deco Example Components', () => {
```

Comment:
this is pretty much a translation of `spec.cy.js` - I used ChatGPT to migrate one to the other